### DISCUSSION TITLE
Customize Branch Name on Github

### DISCUSSION THREAD

**Discussion by ttseng** [codesandbox/codesandbox-client#4127] [General] [answered]
I'm a bit confused by the naming of branches on Github for Github-linked Sandboxes.

For example, when I create a fork of my master branch in CodeSandbox, it has an arbitrary name such as `wandering-dew-dwgh9`

Although I can rename the sandbox to say `Updating UI` or something more descriptive, when I create a PR on Github, it doesn't use either the auto-generated name or the name I've provided; instead the name of the branch is something like `csb-1234567891011`

Is there a way to customize the branch name from within CodeSandbox to make it easier to associate the branch with the particular sandbox the changes were submitted from?

**Comment by garethx** [comment, marked as answer]
Not at the moment, but we're working on some changes to our GitHub flow and that includes making the PR branch name on GitHub the ID of the Sandbox and we will show that in the description, which should help with this.
